### PR TITLE
Get MADX variables into ophyd objects

### PR DESCRIPTION
**Pull request by AbbyGi**
This PR adds support for more models for ophyd classes/objects generation.

### CODE REVIEW

File: sirepo_bluesky/sirepo_ophyd.py [hunk 1]
```diff
@@ -324,3 +324,74 @@ def create_classes(sirepo_data, connection, create_objects=True):
...
+    shadow_config = SimTypeConfig("beamline", "title")
+    madx_config = SimTypeConfig("rpnVariables", "element_name")
+
+    config_dict = {
+        "srw": srw_config,
+        "shadow": shadow_config,
+        "madx": madx_config,
+    }
+
+    for i, el in enumerate(data["models"]["rpnVariables"]):
```
Comment (mrakitin) [inline sirepo_bluesky/sirepo_ophyd.py:348]: `rpnVariables` can be parametrized and passed to the function as a kwarg.

**Review by mrakitin — COMMENTED** [review]
Looks like this use-case can be wrapped into the `create_classes(...)` function. I think an optional kwarg for the field name can replace the key in `enumerate(data["models"][config_dict[sim_type].element_location])`.

**Comment by mrakitin** [comment]
I reworked the code a bit to keep just one function. It can also me compatible with #70.

@AbbyGi, @thomaswmorris, please review the changes.

**Review by jklynch — APPROVED** [review]
Nice test! It all seems good to me.

**Review by AbbyGi — COMMENTED** [review]
Can't approve my own PR but I like the changes.

**Pull request merged**